.github/workflows/lint-test.yaml

```diff
@@ -11,19 +11,35 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.1.0
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Set up Python
```

Comment:
maybe just add a comment above why it's required, as before it was abstracted by having it in the docker image - now the user needs to be aware

Reply:
I added it to the action's readme but I think it makes sense to add a comment here as well.
See https://github.com/helm/chart-testing-action/blob/master/README.md#L24-L26

Reply:
Added a comment